backend/pennclubs/settings/base.py

```diff
@@ -131,7 +131,7 @@
 
 PLATFORM_ACCOUNTS = {
     "REDIRECT_URI": os.environ.get(
-        "LABS_REDIRECT_URI", "http://localhost:8000/api/accounts/callback/"
+        "LABS_REDIRECT_URI", "http://localhost:3000/api/accounts/callback/"
```

Comment:
So setting the port to 3000 will break the login when doing backend-only development, but if you set it to 8000 it will break frontend dev by default. Not sure what we should go with